FAQ.md

```diff
@@ -70,10 +70,10 @@ and guest.
 
 ### What operating systems are supported by Firecracker?
 
-Firecracker supports Linux host and guest operating systems with kernel versions
-4.14 and above, as well as
+Firecracker supports Linux host and guest operating systems as well as
 [OSv](http://blog.osv.io/blog/2019/04/19/making-OSv-run-on-firecraker/) guests.
-The long-term support plan is still under discussion.
+Currently supported host/guest kernel versions can be found in the
```

Comment:
I suggest the following change: Firecracker supports Linux host and guest operating systems with kernel version 4.14 and 5.10. More information supported kernel versions can be found here. Firecracker also runs OSv guests